lib/kube/proxy/forwarder.go

```diff
@@ -562,18 +575,33 @@ func (f *Forwarder) exec(ctx *authContext, w http.ResponseWriter, req *http.Requ
 			events.SessionEventID: sessionID,
 			events.EventUser:      ctx.User.GetName(),
 			events.EventNamespace: f.Namespace,
+			// Note:
+			// - "transmitted" here means from user to pod.
+			// - "received" here means from pod to user.
+			events.DataTransmitted: trackIn.Count(),
+			events.DataReceived:    trackOut.Count() + trackErr.Count(),
```

Comment:
Do you have an example of how these appear in the log output too?

Reply:
This is how these show up:
```json
{
  "code": "T2004I",
  "ei": 5,
  "event": "session.end",
  "namespace": "default",
  "proto": "kube",
  ---> "rx": 16,
  "sid": "b705d5fa-e47a-4bd1-8622-358a42c0f880",
  "time": "2020-04-29T00:25:00.186Z",
  ---> "tx": 1,
  "uid": "58b2dcd1-2ee2-4662-9810-6efd66b19fb3",
  "user": "awly"
}
```

Reply:
Thanks!